### PR TITLE
Corrected spelling mistakes

### PR DESCRIPTION
**Pull request by ravixis29**
I have corrected spelling mistakes in README.md file at two places and one grammatical mistake. I have changed aticities to activities and though to through. 

I believe these changes will improve the overall quality of the article and provide a better experience for readers.

Signed-off-by:  Ravi Kumar <ravixis29@gmail.com>

### CODE REVIEW

**Review by AJThePro99 — APPROVED** [review]
All seems clean and clear.

**Comment by nate-double-u** [comment]
Thanks for this @ravixis29.

Before we can merge this, we do need a DCO signoff. Please take a look a the instructions for how to do that here: https://github.com/cncf/mentoring/pull/1089/checks?check_run_id=16844349189

**Pull request closed without merging**
